/polyx_events: # this ns has to be consistent with launch.py ns(which overwrite node ns)
  event_gen: # this name has to be consistent with node name, not type!
    ros__parameters:
      use_sim_time: False
      latitude: 37.405109067        #[deg]
      longitude: -121.918100758     #[deg]
      ellipsoidal_height: -10.136   #[m]
      roll: 0         #[0.01 deg]
      pitch: 0        #[0.01 deg]
      heading: 2500   #[0.01 deg]
      position_rms: 1  #[cm]
      zupt_rms: 10            #[mm]
      heading_rms: 10        #[0.1 deg]
      flags: 0